security/seccomp-profiles.adoc

```diff
@@ -0,0 +1,77 @@
+[id="seccomp-profiles"]
+= Configuring seccomp profiles
+include::modules/common-attributes.adoc[]
+:context: seccomp-profiles
+
+toc::[]
+
+A container or an OpenShift pod runs a single application that performs one or more well-defined tasks. The application usually requires only a small subset of the underlying operating system kernel APIs.
+Seccomp (secure computing mode) Linux kernel feature can be used to limit the process running in a container to only call a subset of the available system calls. These system calls can be configured by creating profile that is named as seccomp profile.
```

Comment:
```suggestion
Seccomp (secure computing mode) Linux kernel feature can be used to limit the process running in a container to only call a subset of the available system calls. These system calls can be configured by creating a profile that is applied to a container or pod.
```

Reply:
Let's add the information that seccomp profiles are represented as JSON files on disk.